.test-infra/metrics/README.md

```diff
@@ -84,23 +95,31 @@ docker-compose build
 
 # Spinup docker-compose related containers.
 docker-compose up
```

Comment:
I think we should combine these two docker-compose configs or provide a separate config that spins up whole service. Otherwise it will be harder for new developer to figure out the reasons for some dashboards not working.

Reply:
I agree. Combining these two docker-compose files is a good idea.

What do you think about Kubernetes deployments (`beamgrafana-deploy.yaml` and `beamprometheus-deploy.yaml`)? It's a similar situation. Prometheus is an independent entity, but Grafana depends on Prometheus.

Reply:
I believe it is reasonable to have a script that deploys full snapshot. The whole metrics part is rather small setup, so it is beneficial to deploy a whole stack with single command.
We can keeps separate script for each part, but then we should add a single deployment script on top of those.